climada/hazard/tc_tracks_synth.py

```diff
@@ -452,6 +555,16 @@ def _calc_land_decay(hist_tracks, land_geom, s_rel=True, check_plot=False,
     Returns:
         v_rel (dict(category: A)), p_rel (dict(category: (S, B)))
     """
+    
+    if len(hist_tracks) < 100:
```

Comment:
Is this still called when using the global decay parameters?

Reply:
No, see line 221 :)